lib/src/models/usage.dart

```diff
@@ -41,27 +41,34 @@ class PlanUsageBreakdown {
 
 /// Contains main information about usage.
 class Usage {
+
+  /// End date of a billing cycle.
+  DateTime? billingCycleEnd;
+
+  /// Start date of a billing cycle.
+  DateTime? billingCycleStart;
+
+  /// Indicator if the Explorer has access to a 5 min free call.
+  bool? hasFreeCall;
+
   /// Name of the plan.
   String? planName;
 
-  /// Total available minutes in the plan.
-  int? totalMinutes;
+  /// Indicates if the plan is unlimited.
+  bool? planUnlimited;
 
-  /// Total of used minutes in the plan.
-  int? totalMinutesLeft;
+  /// Indicates if the User is Primary.
+  bool primary;
 
-  /// Total amount of primary minutes in the plan. These primary minutes are personal minutes.
-  int? totalPrimaryMinutes;
+  /// Breakdown of the usage by Primary and Secondary users.
+  List<PlanUsageBreakdown>? planUsageBreakdownList;
 
-  /// Total amount of used primary minutes in the plan.
-  int? totalPrimaryMinutesUsed;
+  /// Delay before next free 5 min call.
+  DateTime? timeUntilNextFreeCall;
```

Comment:
`timeUntil` suggests that this is a delta but I think it's the actual time that the next free call becomes available. Maybe `nextFreeCallDate` would be more appropriate. 

Also, I noticed that we sometimes name `DateTime` properties as "TimeStamp" (e.g. `CallSession.endTimeStamp`). It'd be nice if we used a consistent naming convention. My lean is toward "Date"(`Usage.billingCycleEndDate`, `Usage.nextFreeCallDate`, `CallSession.endDate`) as I was using "At" for dates passed as milliseconds since the epoch (`Message.sentAt`).